src/statd/python/yanger/yanger.py

```diff
@@ -497,8 +490,14 @@ def add_container(containers):
 
         # The 'podman ps' command lists ports even in host mode, but
         # that's not applicable, so skip networks and port forwardings
-        networks = container_inspect(container["name"], ("NetworkSettings", "Networks"))
-        if "host" in networks:
+        cont = container_inspect(container["name"])
+        if cont and isinstance(cont, list):
+            cont = cont[0]
+        else:
+            cont = {}
+
+        networks = cont.get("NetworkSettings", {}).get("Networks", {})
```

Comment:
```suggestion
        networks = cont.get("NetworkSettings", {}).get("Networks")
```
I realize I suggested this code :sweat_smile: But the following `if networks ...` makes me think that you want `networks` to be `None` when there are no networks configured, not the empty dict?